extensions/sql-bindings/src/common/azureFunctionsUtils.ts

```diff
@@ -198,12 +198,12 @@ export async function getSettingsFile(projectFile: string): Promise<string | und
 
 /**
  * Retrieves the new function file once the file is created and the watcher disposable
```

Comment:
complete this

Reply:
What do you mean by this?

Reply:
`and the watcher disposable`

the sentence seems incomplete. Or maybe just written wrong (`new function file and watcher disposable once`...?). It doesn't make any sense as it is.

Reply:
Let me know if the updated string ` New azure function file watcher and watcher disposable to be used to watch for changes to the azure function project
` is better.